### ISSUE TITLE
auto check wont check

### ISSUE THREAD

**Issue by Lynchon83** [CGCookie/blender-addon-updater#89]
Hi,

The updater works fine if I manually search for updates but, even if I have enabled auto-check in the preferences panel, it wont launch the function that searches for updates in bitbucket.

I've changed the interval property (inside class `DemoPreferences(bpy.types.AddonPreferences)`) to just one minute in order to see if it looks for any update but nothing happens.

Is there any other seting I should check?

**Comment by schroef** [comment]
Do you see any errors in the console?

**Comment by TheDuckCow** [comment]
Sorry for missing this question @Lynchon83! The other thing is to try restarting blender and then checking if it did the check. The trigger is meant to occur not exactly at the time the interval is reached, but rather at the next eligible time, ie when someone starts a new session. This is to avoid having overhead of a constantly running polling mechanism.

**Comment by TheDuckCow** [comment]
Checkin in - is this still a problem?

**Comment by iMaciaP** [comment]
Hi @TheDuckCow ,

I managed to achieve this feature by adding a "bpy.app.handlers.load_post" handler and call the check now operator. And with that, i call the op whenever a file is loaded, because otherwise old files will remain in "check now for updates" status.

if the setting "check_interval" does not stablish a time interval when will be checked for updates i consider it misspelling.

```
# Optional, to hard-set update frequency, use this here - however, this
# demo has this set via UI properties.
updater.set_check_interval(enabled=True, months=0, days=0, hours=0, minutes=1
```

**Comment by schroef** [comment]
@iMaciaP 
But won't that show the popup dialog each time you open a file and then choose to neglect it?

**Comment by iMaciaP** [comment]
@schroef no, i do not have that behaviour when auto launching the check update

**Comment by schroef** [comment]
Okay I would presume it does, because you call the check on each file load.
If the user doesn't choose to do the upload. Normally that means it shows
again after the timer reaches 0. But since you call it now on each file
load, I was thinking it would then show again. Over and over.
